spec/lib/fingerprint_self_test_spec.rb

```diff
@@ -21,6 +21,13 @@
         expect(db.match_key).not_to be_empty
       end
 
+      it "has valid 'preference' value" do
+          # Reserve values below 0.10 and above 0.90 for users
+          # See xml/fingerprints.xsd
+          expect(db.preference.class).to be ::Float
+          expect(db.preference).to be_between(0.10, 0.90)
```

Comment:
Hmm.  I know the values outside of this range are intended for end-user customization, but won't this mean that if they put their XML in the same place as the ones that this tests, that it will fail?  Seems like it would.  Unsure of the best way to fix this.